app/views/collections/_form.html.erb

```diff
@@ -1,11 +1,19 @@
 <%= simple_form_for [collections, @form], html: {class: 'form-horizontal editor'} do |f| %>
   <div id="descriptions_display">
```

Comment:
@leahvanderjagt has verified that she doesn't want a button, only the text description to display.